erpnext/utilities/doctype/video/video.js

```diff
@@ -2,7 +2,30 @@
 // For license information, please see license.txt
 
 frappe.ui.form.on('Video', {
-	// refresh: function(frm) {
+	refresh: function (frm) {
+		if (frm.doc.provider === "YouTube") {
+			frappe.db.get_single_value("Video Settings", "enable_youtube_tracking").then(value => {
+				if (value) {
+					frm.events.get_video_stats(frm);
+				} else {
+					frm.set_df_property('youtube_tracking_section', 'hidden', true);
+				}
```

Comment:
Instead of refreshing on every load via Form view _(which could be problematic IMO in case a lot of users view this doc frequently)_, can we add a refresh stats frequency in Video Settings to run via scheduler?